cypress/page/personal-input.page.ts

```diff
@@ -0,0 +1,132 @@
+// /<reference types="cypress" />
+
+class PersonalInputPage {
+  private inputPageUrl: string;
+  private firstNameField: string;
+  private lastNameField: string;
+  private emailField: string;
+  private genderField: string;
+  private mobileNumberField: string;
+  private dateOfBirthField: string;
+  private hobbiesField: string;
+  private subjectsField: string;
+  private addressField: string;
+  private submitFormButton: string;
+  private modalInfo: string;
+  private stateField: string;
+  private cityField: string;
+
+  constructor() {
+    this.inputPageUrl = "https://demoqa.com/automation-practice-form";
+    this.firstNameField = "input#firstName";
+    this.lastNameField = "input#lastName";
+    this.emailField = "input#userEmail";
+    this.genderField = "input[name='gender']";
+    this.mobileNumberField = "input#userNumber";
+    this.dateOfBirthField = "input#dateOfBirthInput"
+    this.subjectsField = "#subjectsContainer";
+    this.addressField = "textarea#currentAddress";
+    this.hobbiesField = "div.custom-checkbox";
+    this.stateField = "Select State";
+    this.cityField = "Select City";
+    this.submitFormButton = "#submit";
+    this.modalInfo = ".table > tbody > tr";
+  }
+
+  // 1.  Open page with the Form to fill
+  public openFormPage(): void {
+    cy.visit(this.inputPageUrl);
+  }
+
+  // 2.  Fill hobbiesList array with all available hobbies
+  private enterHobbies(hobbiesList: string[]): void {
+    for (let hobbie in hobbiesList) {
+      cy.get(this.hobbiesField)
+        .contains("label", hobbiesList[hobbie])
+        .siblings()
+        .click({ force: true });
```

Comment:
You could have used cy.check()

Reply:
I will fix it for the next PR.   Thanks for the advise.